src/PluginWrapper.php

```diff
@@ -85,7 +85,7 @@ public function postPackage(PackageEvent $event)
     {
         $packages = $this->getPackagesFromEvent($event);
         $key = $this->getUniqueNameFromPackages($packages);
-        if ($this->preservers[$key]) {
+        if (!empty($this->preservers[$key])) {
```

Comment:
Shouldn't it rather be `if ($this->preservers[$key] && !empty($this->preservers[$key]))`?

Reply:
I don't think that would fix the issue. !empty first will both verify the key is present and validate it contains something. The original patch fixes the issue on my end.

Reply:
Perhaps `!empty($this->preservers) && $this->preservers[$key]` might make better sense.

Reply:
When debugging, I'm seeing preserve-paths get called to validate itself. But ->preservers is null at that point.

Reply:
Ah yeah! You are right. Just read the docs again.

> `empty()` is essentially the concise equivalent to `!isset($var) || $var == false`

So the original patch should be fine indeed.

Reply:
With the exception that the problem here is ->preservers is null when the defect is triggered. Its not that the array has null values, but the the array does not exist yet.

Reply:
`empty()` will not trigger an error if the variable doesn't exist, this means that `!empty($this->preservers[$key])` will not throw a warning, even if `$this->preservers` is null.

Reply:
Agreed, I've confirmed this PR resolves the issue. I sent an email to Johannes Haseitl to get his attention to the PR. Can anyone else try to reach out and get some traction on this?

Reply:
I've also tried to get into contact with @derhasi on the Drupal Slack server to no avail.

Can we push this through without his approval? This change doesn't seem dangerous.

Reply:
I‘m sorry I did not react to this. I am currently not really maintaining any opensource projects 😞.

This change looks pretty straightforward. Thanks for your fix @Arkener